CONTENT/about.markdown

```diff
@@ -1,6 +1,6 @@
 [//]: # (@TODO - make this file nice)
```

Comment:
One thing first - before you start changing stuff and updating the pull request, it would be great to discuss it here in the comments below and come to consensus. That would be great.